### ISSUE TITLE
CP56Time2a to/from MsTimestamp convert bug

### ISSUE THREAD

**Issue by salfetko** [mz-automation/lib60870#73]
Functions CP56Time2a_setFromMsTimestamp and CP56Time2a_toMsTimestamp in cpXXtime2a.c not reversibly:
` CP56Time2a_setYear(self, tmTime.tm_year);` vs `tmTime.tm_year = CP56Time2a_getYear(self) + 100;`

**Comment by mzillgith** [comment]
Yes. You are right. This works only for years between 2000 and 2099. Yet I don't know how to solve it as the CP56Time2a allows only to store year values between 0 and 99. So either the function has to assume that the year is in a specific range (as it does now) or require the user to specify the correct range.
